### DISCUSSION TITLE
how do I get it to run at close to 100% performance.

### DISCUSSION THREAD

**Discussion by msjsc001** [openai/whisper#815] [Q&A]
![image](https://user-images.githubusercontent.com/93178788/210845192-8faa43fd-8171-4367-8207-294fbc9b4da1.png)
The GPU I'm using is a 3080ti and it only runs at 40% of the GPU, how do I get it to run at close to 100% performance.
I'm using GPU A2000 and it's getting close to 100% performance.
This is the statement I am currently using `whisper mnt/mp3/A.mp3 --model large-v1 --language English --output_dir mnt/mp3/`

**Comment by turnkit** [comment]
I have the same question.  I think mainly though there are some obvious issues -- on initial startup the media file is read into RAM and the GPU is not active at that time.  So to keep the GPU active you must decrease time spent waiting for loading.  Suggest maxing out RAM, ensure media files are on very fast storage.   It's possible then even that software like Primocache or others might reduce the loading times for the media files, especially if you have maxed out your system RAM.

See: https://alternativeto.net/software/super-cache/

**Comment by EtienneAb3d** [comment]
One possibility is to load N copies of Whisper and process N files in parallel.

**Reply by msjsc001** [reply]
> One possibility is to load N copies of Whisper and process N files in parallel.

I have tested the 2 Whisper running at 3080ti or 3090 approach (2 Whisper video memory requires 24G) and 2 Whisper does boost performance and memory usage to 100%.
It would be nice if there was a way to run a single Whisper at full capacity as well, so I could run it on a low graphics memory card.
